Include/cpython/initconfig.h

```diff
@@ -179,6 +179,7 @@ typedef struct PyConfig {
     int use_frozen_modules;
     int safe_path;
     int int_max_str_digits;
+    int cpu_count;
```

Comment:
Maybe we should start adding version to this complex structure:

```suggestion

    // Python 3.13
    int cpu_count;
```

Reply:
Would it not be better to document the feature instead of the version? Using simple Git tricks, it is very easy to find out which Python version a field was added in; I don't see the value in duplicating Git log in the source code :)

Reply:
All PyConfig must be documented at: https://docs.python.org/dev/c-api/init_config.html

See my notes on adding a new member to PyConfig: https://pythondev.readthedocs.io/pyconfig.html

Reply:
> All PyConfig must be documented at: https://docs.python.org/dev/c-api/init_config.html

Yes, so IMO:

- remove the superfluous `// Python 3.13` comment from initconfig.h
- add an entry accompanied with a `.. versionadded` to [Doc/c-api/init_config.rst](https://github.com/python/cpython/blob/main/Doc/c-api/init_config.rst)

Reply:
Sure, in the doc, versionadded must be used, obviously.

> remove the superfluous // Python 3.13 comment from initconfig.h

I think that it would nice to *also* document the version in PyConfig structure, in addition to the init_config.rst doc. But I will not insist :-)

Reply:
Will a `// Python 3.13` comment make sense on its own? IMO: no. How would a random contributor interpret it? :) How will you or I interpret it in five years time? Isn't it better to leave the version control to the version control system? :)

Reply:
... so you can count me -1 on sprinkling the source code with `// Python x.y` comments :)

Reply:
Ok, let's remove the "// Python 3.13" comment.